module/enrichers.mjs

```diff
@@ -636,6 +637,95 @@ async function enrichSave(config, label, options) {
 
 /* -------------------------------------------- */
 
+/**
+ * Enrich an item use link to roll an item on the selected token.
+ * @param {string[]} config            Configuration data.
+ * @param {string} [label]             Optional label to replace default text.
+ * @returns {Promise<HTMLElement|null>}         An HTML link if the item link could be built, otherwise null.
+ *
+ * @example Use an item from a Name:
+ * ```[[/item Heavy Crossbow]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Heavy Crossbow
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a UUID:
+ * ```[[/item Actor.M4eX4Mu5IHCr3TMf.Item.amUUCouL69OK1GZU]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Bite
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a "Name UUID":
+ * ```[[/item Actor.Akra (Dragonborn Cleric).Item.Mace]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Mace
+ * </a>
+ * ```
+ *
+ * @example Use an Item from a Relative UUID:
+ * ```[[/item .amUUCouL69OK1GZU]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Bite
+ * </a>
+ * ```
+ */
+async function enrichItem(config, label) {
+  const givenItem = config.values.join(" ");
+  // If config is a UUID
+  const itemUuidMatch = givenItem.match(/^(?<synthid>Scene\.\w{16}\.Token\.\w{16}\.)?(?<actorid>Actor\.\w{16})(?<itemid>\.?Item(?<relativeId>\.\w{16}))$/);
+  if (itemUuidMatch) {
+    const ownerActor = itemUuidMatch.groups.actorid.trim();
+    if (!label) {
+      if ((await fromUuid(givenItem)) == null) {
+        console.warn(`Item not found while enriching ${givenItem}.`);
+        return null;
+      } else label = (await fromUuid(givenItem)).name;
+    }
+    return createRollLink(label, { type: "item", rollItemActor: ownerActor, rollItemUuid: givenItem });
+  }
+
+  // If config is a relative Id/Name
+  const relativeIdMatch = givenItem.match(/^\.\w{16}$/);  // .amUUCouL69OK1GZU
+  const copiedIdMatch = givenItem.match(/^\w{16}$/);      // amUUCouL69OK1GZU
```

Comment:
This doesn't seem to be used anywhere other than the `if` statement and `[[/item amUUCouL69OK1GZU]]` isn't a valid format for the enricher.

Reply:
Why wouldnt that be a valid format? If a user clicks on the book icon of an item to copy its id, then this is the format they get so I thought it was important to account for. 
Essentially, if the `givenItem` is 16 characters, first check it as an id, if it doesnt match then it is treated as a name